### PR TITLE
Adds Llama 3.2 example on Modal with a fun experiment

### PR DESCRIPTION
**Pull request by erik-dunteman**
# What does this PR do?

We add a new third party recipe for [Modal](https://modal.com), a serverless python platform that allows people to easily run experiments on Llama.

This is a continuation of the blog:
[Beat GPT-4o at Python by searching with 100 dumb LLaMAs](https://modal.com/blog/llama-human-eval)

### Testing

Given user has Llama 3.2 license approval on Huggingface, has their HF_TOKEN configured as a Modal secret, and has logged into Modal CLI, the included end-to-end script has been ran successfully:

`run_e2e.sh`

These steps have been included in the README.

## Before submitting
- [ ] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [x] Did you read the [contributor guideline](https://github.com/facebookresearch/llama-recipes/blob/main/CONTRIBUTING.md#pull-requests),
      Pull Request section?
- [ ] Was this discussed/approved via a Github issue? Please add a link
      to it if that's the case.
- [x] Did you make sure to update the documentation with your changes?  
- [ ] Did you write any new necessary tests?

Thanks for contributing 🎉!

### CODE REVIEW

**Comment by erik-dunteman** [comment]
PR submitted by suggestion of @charlesfrye

File: recipes/3p_integrations/modal/many-llamas-human-eval/README.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+*If we run enough Llama models in parallel, can they outperform GPT-4o on HumanEval?*
+
+It seeks to increase model performance not through scaling parameters, but by scaling compute time.
+
+### Technical Blog
+
+This experiment built by the team at [Modal](https://modal.com), and is described in the following blog post:
+
+[Beat GPT-4o at Python by searching with 100 dumb LLaMAs](https://modal.com/blog/llama-human-eval)
```
Comment (HamidShojanazeri) [inline recipes/3p_integrations/modal/many-llamas-human-eval/README.md:13]:
```suggestion
[Beat GPT-4o at Python by searching with 100 small Llamas](https://modal.com/blog/llama-human-eval)
```

**Review by HamidShojanazeri — APPROVED** [review]
LGTM, one minor comment on the messaging added inline.

**Comment by erik-dunteman** [comment]
Just approved, thanks for keeping PR open!

**Pull request merged**